src/cli/index.ts

```diff
@@ -109,14 +112,14 @@ async function samplesFromDirectory(dataDir: string, topLevelRefs: string[] | un
             }
 
             if (file.endsWith(".url") || file.endsWith(".json")) {
-                // FIXME: Why do we include the URI here?
                 sourcesInDir.push({
+                    kind: "json",
                     name,
-                    uri: fileOrUrl,
```

Comment:
@schani I have no idea if this is used. Seems like it was not.

Reply:
Weird.  It wasn't even in the type definition.